src/dao/pages/MultiSigDAODashboard/MultiSigDAODashboard.tsx

```diff
@@ -1,19 +1,23 @@
 import { TokenBalance, useAccountTokenBalances, usePairedWalletDetails } from "@dex/hooks";
-import { useDAOs } from "@dao/hooks";
+import { useContract, useDAOs } from "@dao/hooks";
 import { Member, MultiSigDAODetails } from "@dao/services";
 import { Outlet, useParams } from "react-router-dom";
 import { isNil, isNotNil } from "ramda";
 import { DAODashboard } from "../DAODashboard";
 import { MultiSigDAODetailsContext } from "./types";
 
 export function MultiSigDAODashboard() {
-  const { accountId: daoAccountId = "" } = useParams();
-  const daosQueryResults = useDAOs<MultiSigDAODetails>(daoAccountId);
+  const { accountId = "" } = useParams();
+  const daoAccountIdQueryResults = useContract(accountId);
```

Comment:
If accountId is an EVM address and no longer an Account Id, appropriately rename the variable to explicitly highlight this change.

Reply:
accountId is the EVMAddress for MultiSig but for FT and NFT DAOs it is AccountId. so with this PR I have not changed the name of the variable. I wull update it with my next PR, in which I am making changes in the FT and NFT DAOs.